Blog: EU Digital sovereignty

kharonsec opened the pull request:
Article detailing the EU digital sovereignty problem, how the EU became dependant on U.S technology, what are the risks of such a dependence and what could be the path forward.

kharonsec committed:
Add files via upload

kharonsec committed:
Merge pull request #1 from kharonsec/eu_digital_sovereignty_Final

semi-Final version, still open to changes

hazcod commented on content/blog/the-myth-of-digital-sovereignty.md:
```diff
@@ -0,0 +1,83 @@
+# EU Digital Sovereignty at Risk
+
+## Introduction
+
```
I lack an intro on why sovereignty is an issue, and how we came up to this point.
Why did we move massively to US silicon valley clouds?

hazcod commented on content/blog/eu_digital_sovereignty.md:
```diff
@@ -0,0 +1,83 @@
+
+## What Are US Companies Putting in Place to Reassure Europe of Their Sovereignty?
+
+The biggest US companies providing cloud services in Europe have come forward with their own "sovereign" solutions to try and reassure Europeans that their data is safe and remains under their control.
```
Add a segment on what an "easy" solution is to fix the sovereignty issue, and how US companies are doing this.

hazcod commented on content/blog/The-Myth-of-Digital-Sovereignty.md:
```diff
@@ -0,0 +1,83 @@
+
+The biggest US companies providing cloud services in Europe have come forward with their own "sovereign" solutions to try and reassure Europeans that their data is safe and remains under their control.
+
+### Microsoft
```
Add reference to Azure and Office365

hazcod commented on content/blog/eu_digital_sovereignty.md:
```diff
@@ -0,0 +1,83 @@
+
+Microsoft has launched sovereign solutions which will ensure that all European data stays in Europe and is controlled by Microsoft employees in Europe. It will also include an encryption controller operated by the customer, meaning the cloud provider will not have the decryption key.
+
+### AWS
```
Amazon Web Services (AWS)*

hazcod commented on content/blog/eu_digital_sovereignty.md:
```diff
@@ -0,0 +1,83 @@
+
+### Microsoft
+
+Microsoft has launched sovereign solutions which will ensure that all European data stays in Europe and is controlled by Microsoft employees in Europe. It will also include an encryption controller operated by the customer, meaning the cloud provider will not have the decryption key.
```
What are these "sovereign solutions"?

hazcod commented on content/blog/eu_digital_sovereignty.md:
```diff
@@ -0,0 +1,83 @@
+
+### AWS
+
+AWS is about to launch its version of a dedicated cloud for Europe, with infrastructure and personnel from Europe having access to and control of the data stored in Europe.
```
How will it work? Hardware supplied to the DC? And how are updates applied?

hazcod commented on content/blog/The-Myth-of-Digital-Sovereignty.md:
```diff
@@ -0,0 +1,83 @@
+
+No matter where the data resides, as long as it is stored on hardware owned by an American company or one of its subsidiaries, that data falls under U.S. jurisdiction. If the U.S. government requests access, the company is legally bound to comply, or face severe consequences such as heavy fines, loss of federal contracts, or even exclusion from doing business in the United States.
+
+Even Microsoft admitted before the French Senate that it could not guarantee that, if the U.S. government demanded access to data belonging to French citizens and stored in France, it would be able to block that request.
```
Maybe add a reference

hazcod commented on content/blog/eu_digital_sovereignty.md:
```diff
@@ -0,0 +1,83 @@
+
+All these initiatives share one fundamental flaw: under the CLOUD Act, geography does not matter—jurisdiction does.
+
+No matter where the data resides, as long as it is stored on hardware owned by an American company or one of its subsidiaries, that data falls under U.S. jurisdiction. If the U.S. government requests access, the company is legally bound to comply, or face severe consequences such as heavy fines, loss of federal contracts, or even exclusion from doing business in the United States.
```
Is this really a hard statement in regards to the US owned hardware?
What about GCP datacenters operated by Thales in France? Maybe they own the GCP hardware? (bought it/leased it?)

hazcod commented on content/blog/eu_digital_sovereignty.md:
```diff
@@ -0,0 +1,83 @@
+
+Microsoft claims that such a request would need to be "legitimate." But how much trust can Europe really place in a company whose ultimate goal is to maximize profit, and whose current leadership, while perhaps law-abiding today, could one day change?
+
+What happens if the United States drifts toward a more authoritarian state, and someone less concerned with legal boundaries takes control of Microsoft? Who will then guarantee that European data remains protected and only accessed under legitimate circumstances?
```
(more) towards :P

hazcod commented on content/blog/eu_digital_sovereignty.md:
```diff
@@ -0,0 +1,83 @@
+
+The sovereignty problem does not stop at "Who has access to my data?" but also relies on the security of the hardware on which it is stored.
+
+A recent study by KU Leuven and the University of Birmingham has shown a critical flaw in Intel's and AMD's processor chips, allowing any person with physical access to the server to steal data in an almost invisible way.
```
Add a small example of how this would play out. e.g. if you host EU data on a US datacenter.
I'm also missing how confidential compute could play into this, e.g. AWS NITRO instances.

hazcod commented on content/blog/eu_digital_sovereignty.md:
```diff
@@ -0,0 +1,83 @@
+
+Europe faces a double dilemma: it relies on foreign infrastructure on one side and on foreign hardware on the other side, both of which undermine Europe's ambition to be fully digitally sovereign.
+
+Some initiatives are emerging, such as the Rhea-1 supercomputer chip developed in France by SiPearl, OVHCloud's Sovereign Solution, and NumSpot, another European cloud provider. Yet these initiatives cannot yet compete with giants like Google, Microsoft, or Amazon.
```
Also good to mention the EU initiative, which is veeeery slowly coming

hazcod commented on content/blog/eu_digital_sovereignty.md:
```diff
@@ -0,0 +1,83 @@
+
+To change this, Europe must:
+
+- **Invest in European companies** developing products aimed at making Europe more sovereign.
```
Mention the difference in technologic advancements between silicon valley and EU.

kharonsec committed:
Update eu_digital_sovereignty.md

Added more content and multiple sources for the main claims

kharonsec committed:
Update eu_digital_sovereignty.md

Updated all requested sections.

hazcod commented on content/blog/eu_digital_sovereignty.md:
```diff
@@ -0,0 +1,83 @@
+
+Data sovereignty refers to the principle that data is subject to the laws of the country in which it resides.
+
+For individuals, this subject might seem trivial—after all, everyone shares data online at their own discretion. However, the stakes increase drastically when we consider data collected by governments or public institutions. This includes medical records, financial information, and government communications.
```
This is too abstract. I'd give an example on what this means @kharonsec.
US government can extract your Google Drive files, Apple iCloud (without advanced datata protection), OneDrive, emails, ...

hazcod commented on content/blog/eu_digital_sovereignty.md:
```diff
@@ -0,0 +1,83 @@
+# EU Digital Sovereignty at Risk
```
@kharonsec I always try to come up with a snarky title. Also, you're missing the blog post metadata.
Try looking at other blog posts in my repo for an example.

hazcod commented on content/blog/eu_digital_sovereignty.md:
```diff
@@ -1,4 +1,4 @@
-# The Myth of Digital Sovereignty
++++ date = "2025-10-10T16:30:00+02:00 " title = "The Myth of Digital Sovereignty" tags = ["EU","digital sovereignty","AWS","Google","Cloud","azure","cloud act","aws nitro","gaia","OVH","Rhea-1"] description = "The situation of the digital sovereignty in the European Union, it's reasons and the path forward." layout = "blog" draft = false +++
```
@kharonsec Can you use multiline please?

kharonsec committed:
Update EU Digital Sovereignty article with new sections on practical examples, confidential compute, and Silicon Valley vs EU tech gap

kharonsec committed:
Final updates to EU Digital Sovereignty article

hazcod commented on content/blog/eu_digital_sovereignty.md:
```diff
@@ -0,0 +1,86 @@
++++ date = "2025-10-10T16:30:00+02:00 " title = "The Myth of Digital Sovereignty" 
+tags = ["EU","digital sovereignty","AWS","Google","Cloud","azure","cloud act","aws nitro","gaia","OVH","Rhea-1"] 
```
Still not fully multiline @kharonsec

kharonsec committed:
Fix front matter formatting and address reviewer feedback

- Fixed YAML front matter to be properly multiline
- Added concrete examples of US government data extraction (Google Drive, iCloud, OneDrive, emails)
- Added reference to Azure and Office365 in Microsoft section
- Improved introduction with more specific examples

kharonsec committed:
Rename eu_digital_sovereignty.md to The-Myth-of-Digital-Sovereignty.md

hazcod committed:
chore(blog): sovereignty fixes

Add proposed changes to blog post.

kharonsec committed:
Normalize filename: ensure blog post uses .md extension (fix post-pull mismatch)

kharonsec committed:
chore(blog): remove lingering TODO from article

kharonsec committed:
feat(blog): elaborate on confidential computing technical details

- Added comprehensive explanation of disk encryption, RAM encryption, secure boot, and TPM binding
- Detailed how hardware-based security creates isolated execution environments
- Maintained focus on legal jurisdiction limitations vs technical protection
- Enhanced technical depth while preserving core sovereignty argument
